akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala

```diff
@@ -212,9 +212,9 @@ private[cluster] class ClusterHeartbeatSender extends Actor {
     val nextHB = selfHeartbeat()
     state.activeReceivers.foreach { to =>
       if (failureDetector.isMonitoring(to.address)) {
-        if (verboseHeartbeat) logDebug("Heartbeat to [{}]", to.address)
+        if (verboseHeartbeat) logDebug("Heartbeat #{} to [{}]", nextHB.sequenceNr, to.address)
       } else {
-        if (verboseHeartbeat) logDebug("First Heartbeat to [{}]", to.address)
+        if (verboseHeartbeat) logDebug("First Heartbeat (#{}) to [{}]", nextHB.sequenceNr, to.address)
```

Comment:
```suggestion
        if (verboseHeartbeat) logDebug("First Heartbeat #{} to [{}]", nextHB.sequenceNr, to.address)
```

Reply:
The ClusterHeartbeatSender has a single seqNr counter that is used when sending to all of its receivers. The receivers may change over time, so it's not always seqNr 0 for the first hearteat. I just removed the `( )`